src/docs/sdk/features.mdx

```diff
@@ -231,6 +231,57 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
 
+### HTTP Client Errors
+
+The SDK automatically captures HTTP Client errors and sends them to [sentry.io](https://sentry.io).
+
+The HTTP Client integration should have 3 configuration options:
+
+- `captureFailedRequests` defaults to `false` due to PII reasons.
+  - The SDK will only capture HTTP Client errors if it is enabled.
+- `failedRequestStatusCodes` defaults to `500 - 599`, this configuration option accepts a `List` of `HttpStatusCodeRange` which is a range of HTTP status code -> `min` to `max` or a single `status_code`.
```

Comment:
`m`: A range can also be from 501 to 501. It would simplify the `HttpStatusCodeRange` class. Also noted that here https://github.com/getsentry/sentry-java/pull/2287/files#r998089883.
```suggestion
- `failedRequestStatusCodes` defaults to `500 - 599`, this configuration option accepts a `List` of `HttpStatusCodeRange` which is a range of HTTP status codes from `min` to `max`.
```

Reply:
I disagree, just more boilerplate/confusing for the user, let's wait for the opinion of others.